### ISSUE TITLE
You can see your arms with first person legs enabled 

### ISSUE THREAD

**Issue by conneath** [TF2V/TF2Vintage#30]
you can barely see your arms as little twigs
kinda spooky
i also barely managed to see my head i think but i cant replicate that now
https://conneath.gq/repo/4armsmorefun.mp4

**Comment by TF2V** [comment]
Laughed at this one.
Least I know the legs work now, kind of.

**Comment by AbdallatifAlAfandi** [comment]
Also, when you disguise as spy you can also see the paper mask on your screen

**Comment by TF2V** [comment]
Yeah, it's a very WIP implementation of first person legs, so I'm not surprised it has so many issues.
I'm going to bandage this by disabling it for now as a debug item.
